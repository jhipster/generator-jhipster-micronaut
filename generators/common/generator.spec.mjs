import expect from 'expect';

import { helpers, lookups } from '#test-utils';

describe('SubGenerator common of micronaut JHipster blueprint', () => {
  describe('run', () => {
    let result;
    before(async function () {
      result = await helpers
        .create('jhipster:common')
        .withOptions({
          reproducible: true,
          defaults: true,
          blueprint: 'micronaut',
        })
        .withLookups(lookups)
        .run();
    });

    it('should succeed', () => {
      expect(result.getStateSnapshot()).toMatchSnapshot();
    });
  });
});
