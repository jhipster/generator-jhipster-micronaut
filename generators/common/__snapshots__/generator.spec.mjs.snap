// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SubGenerator common of micronaut JHipster blueprint run should succeed 1`] = `
Object {
  ".editorconfig": Object {
    "stateCleared": "modified",
  },
  ".gitattributes": Object {
    "stateCleared": "modified",
  },
  ".gitignore": Object {
    "stateCleared": "modified",
  },
  ".husky/pre-commit": Object {
    "stateCleared": "modified",
  },
  ".lintstagedrc.js": Object {
    "stateCleared": "modified",
  },
  ".prettierignore": Object {
    "stateCleared": "modified",
  },
  ".prettierrc": Object {
    "stateCleared": "modified",
  },
  ".yo-rc.json": Object {
    "stateCleared": "modified",
  },
  "README.md": Object {
    "stateCleared": "modified",
  },
  "README.md.jhi": Object {
    "state": "modified",
  },
  "package.json": Object {
    "stateCleared": "modified",
  },
  "sonar-project.properties": Object {
    "stateCleared": "modified",
  },
}
`;
